src/components/Shared/PreTransactionModal.tsx

```diff
@@ -211,7 +198,7 @@ export default class PreTransactionModal extends React.Component<IProps, IState>
             </div>
             <div className={css.proposalInfo}>
               <div className={css.proposalTitle}>
-                <strong>{proposal.title || "[no title]"}</strong>
+                <strong>{proposal.title || proposal.id}</strong>
```

Comment:
Isn't proposal.id a hash? Is that better to show than "No title"? Could also be: "Title unavailable"

Reply:
Can we please combine the two but only using part of the id?
Like this - [No title 0xc839...8eee]

Reply:
Ok